tests/test_util.py

```diff
@@ -212,18 +227,16 @@ def test_complex_conditions_expr(self):
         assert complex_column_expr(tuplify(['foo', ['b', 'c'], 'd']), body.copy()) == '(foo(b, c) AS d)'
         assert complex_column_expr(tuplify(['foo', ['b', 'c', ['d']]]), body.copy()) == 'foo(b, c(d))'
 
-        # we may move these to special Snuba function calls in the future
-        assert complex_column_expr(tuplify(['topK', [3], ['project_id']]), body.copy()) == 'topK(3)(project_id)'
```

Comment:
I'm just reminded here. this refactor does remove the ability to express non-nested function calls ie. `foo(bar)(baz)` I _think_ the topK thing was the only place where we used a function returning a function, and we have a special case for that now. But I guess I'm just adding a note in case you've maybe seen any other uses like this.